docs/post-upgrade/update-deprecated-sn-connector.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[cases-update-deprecated-sn-connector]]
+== Update a deprecated ServiceNow connector
+
+{sn} ITSM and SecOps connectors created in 7.15 or earlier are marked as deprecated after you upgrade to 7.16. Deprecated connectors have a yellow icon after their name and display a warning message when you select them.
+
+[role="screenshot"]
+image::images/cases-deprecated-sn-connector.png[]
+
+IMPORTANT: Deprecated connectors will continue to function with the rules and cases they were added to, but you cannot assign a deprecated connector to a new rule or case. If you try to do so, you will be prompted to update the connector or create a new one.
+
+=== Prerequisites
+Before creating a new {sn} ITSM or SecOps connector, you must complete the following within your {sn} instance:
+
+* Install Elastic for ITSM or Elastic for Security Operations (SecOps) from the https://store.servicenow.com/sn_appstore_store.do#!/store/home[ServiceNow Store].
+* Create a {sn} integration user and assign it the appropriate roles. 
+* Create a CORS rule. 
+
+*Create a {sn} integration user*
+
+To ensure authenticated communication between Elastic and {sn}, create a {sn} integration user and assign it the appropriate roles.
+
+. Go to System *Security -> Users*. 
+. Click *New*. 
+. Complete the form. 
+. Assign the integration user the following roles:  
+** `import_set_loader`
+** `import_transformer`
+** `personalize_choices`
+** `sn_si.basic`
+** `x_elas2_sir_int.integration_user`
+. Click *Submit* to create the user.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between the Elastic and {sn}. To create a CORS rule:
+
+. Go to *System Web Services -> CORS Rules*.
+. Click *New*.
+** *REST API*: Choose `x_elas2_sir_int/elastic_api`.
+** *Domain*: Enter the Kibana URL.
+. Go to the *HTTP methods* tab and select *GET*. This chooses the GET HTTP method.
+. Click *Submit* to create the rule.
+
+=== Update a deprecated {sn} connector
+
+Follow these steps:
+
+. Go to *Cases -> Edit external connection*.
+. From the *Incident management system list*, select the deprecated connector to open the connector Edit flyout.
+. Click *Update <connector name>*.
+. In the warning message, click *Update this connector*.
+. Complete the guided steps in the Edit connector flyout.
+.. Install Elastic for ITSM or Elastic for Security Operations (SecOps) from the https://store.servicenow.com/sn_appstore_store.do#!/store/home[ServiceNow Store].
```

Comment:
We can use the following links for each app: 

ITSM: https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88
SecOps: https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f